README.md

```diff
@@ -1,2 +1,2 @@
-# wildnwooley
+# Wild and Wooley
```

Comment:
Honestly anyone reading this will think I'm nuts but he honestly told me to go blah blah blah. When I questioned him on it he said just write some thing here so here something.